images/prow-tests/in-gvm-env.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "${HOME}/.gvm/scripts/gvm"
+
+set -e
+set -x
+
+if [[ ! -v GO_VERSION ]]; then
+  echo "--- FAIL: need GO_VERSION defined for in-gvm-env.sh"
+  exit 1
+fi
+gvm use "${GO_VERSION}"
+# Get our original Go directory back into GOPATH
+pushd /go || exit 5
```

Comment:
Why using 5 - `I/O error`?

Reply:
Who said that's what it means?

Reply:
I checked from https://www.cyberciti.biz/faq/linux-bash-exit-status-set-exit-statusin-bash/.
Actually it does not matter to me as long as it's a non-zero issue, just being curious.

Reply:
https://en.wikipedia.org/wiki/Exit_status says there are various "conventions" and no certain meanings.

I just picked an arbitrary non-1 number, but 2 is also an arbitrary non-1 number :)